main/inc/lib/sessionmanager.lib.php

```diff
@@ -5274,7 +5274,7 @@ public static function getAllUsersFromCoursesFromAllSessionFromStatus(
         }
 
         $courseList = CourseManager::get_courses_followed_by_drh($userId, DRH);
-        $courseConditions = ' AND 1 <> 1';
+//        $courseConditions = ' AND 1 <> 1'; //Why?
```

Comment:
I believe it has a default value to avoid generating a notice when using it in the SQL concat. It should be an empty string from the start (declaration a few lines above). I'm changing that while integrating your PR.